src/tmpfiles/tmpfiles.c

```diff
@@ -81,8 +81,8 @@ typedef enum ItemType {
         CREATE_DIRECTORY = 'd',
         TRUNCATE_DIRECTORY = 'D',
         CREATE_SUBVOLUME = 'v',
-        CREATE_SUBVOLUME_INHERIT_QUOTA = 'q',
-        CREATE_SUBVOLUME_NEW_QUOTA = 'Q',
+        CREATE_SUBVOLUME_WITH_QUOTA = 'q',
```

Comment:
renamed to make the following read a bit clearer with shared logic around `append_or_force`

https://github.com/systemd/systemd/blob/ad00110fa055101eff64c8329af2d75cbfd35e49/src/tmpfiles/tmpfiles.c#L1615-L1618